caravel/viz.py

```diff
@@ -14,7 +14,6 @@
 import uuid
 from collections import OrderedDict, defaultdict
 from datetime import datetime, timedelta
-
```

Comment:
I'll merge anyways, but the idea for that blank line is to differentiate between standard lib imports and external ones. It matters most in large modules with lots of imports...